xrobot/render_engine/model.cpp

```diff
@@ -45,6 +45,16 @@ ModelData::~ModelData() {
     }
 }
 
+void ModelData::BoneTransform(float time, float duration,
```

Comment:
Maybe it is time to gradually add comments, starting from functions and classes that users may need to know to use xrobot.